app/packages/grits-net-meteor/client/templates/grits_search.coffee

```diff
@@ -350,12 +350,12 @@ Template.gritsSearch.onRendered ->
     # update the ajax-loader
     isUpdating = Session.get(GritsConstants.SESSION_KEY_IS_UPDATING)
     # do not show the filter spinner if the overlay isLoading
-    if isUpdating && !Template.gritsOverlay.isLoading()
+    if isUpdating and not Session.get 'loading'
```

Comment:
We could potentially rewrite this as a three liner:
```
filtering = isUpdating and not Session.get 'loading'
$('#applyFilter').prop('disabled', filtering)
Session.set 'filtering', filtering
```

but that might be a little nit picky.  I would say it's your call @jgoley